Core/Core/ActAsUser/ActAsUserViewController.swift

```diff
@@ -101,9 +101,11 @@ public class ActAsUserViewController: UITableViewController {
     }
 
     @IBAction func actAsUserPressed(_ sender: UIButton) {
-        guard let domain = self.domainTextField.text, let userID = self.userIDTextField.text else {
+        guard let domain = domainTextField.text, let userID = userIDTextField.text else {
             return
         }
+        domainTextField.resignFirstResponder()
+        userIDTextField.resignFirstResponder()
```

Comment:
nice!. This works well.  Not sure if you knew that you can call `view.endEditing(true)` as well when there are multiple responders.

Reply:
Oh whoa I did not know about `endEditing`. I like that way better. Updated.